test/ssltest.c

```diff
@@ -769,7 +823,7 @@ static void sv_usage(void)
     fprintf(stderr, " -no_dhe       - disable DHE\n");
 #endif
 #ifndef OPENSSL_NO_EC
-    fprintf(stderr, " -no_ecdhe     - disable ECDHE\n");
+    fprintf(stderr, " -no_ecdhe     - disable ECDHE (obsolete)\n");
```

Comment:
Well, actually, _disabling_ ecdhe isn't obsolete, it's useful for testing RSA kx, and at least one test thinks it's using it... But this was wrong already, so just remove the "obsolete" and stick a `TODO(openssl-team): no_ecdhe was broken by auto ecdh. Make this work again.` in there.